### DISCUSSION TITLE
Is their a way

### DISCUSSION THREAD

**Discussion by Nicc0110** [coder/code-server#3201] [Q&A] [answered]
For the auth is their a way i could add 2fa or something to it?

**Comment by nooobcoder** [comment, marked as answer]
I don't know if there is any such method to add 2FA to `code-server`.
@Mcboy13  there is an issue that was created regarding the same topic #1625 #1556 You can have a look at that issue.
The issue is now closed
@code-asher @jsjoeio Please have a look at this discussion and maybe change mind to work on 2FA as a milestone of the project. @Mcboy13 nice suggestion btw to secure the dev environment.

**Comment by jsjoeio** [comment]
If you run code-server with `--link`, it will ask you to authorize our GitHub app. And on GitHub, you can enable 2FA.

Here's a [demo](https://www.youtube.com/watch?v=MdIEiZpg5vQ).
